lib/gui/modules/drive-scanner.js

```diff
@@ -111,8 +111,21 @@ const scan = () => {
     },
     usbboot: {
       files: {
-        'bootcode.bin': USBBOOT_BOOTCODE_BIN_BUFFER,
-        'start.elf': USBBOOT_START_ELF_BUFFER
+        'bootcode.bin': readBlob('usbboot', 'bootcode.bin'),
+        'config.txt': readBlob('usbboot', 'config.txt'),
+        'start_cd.elf': readBlob('usbboot', 'start_cd.elf'),
+        'fixup_cd.dat': readBlob('usbboot', 'fixup_cd.dat'),
+        'dt-blob.bin': readBlob('usbboot', 'dt-blob.bin'),
+        'cmdline.txt': readBlob('usbboot', 'cmdline.txt'),
+        'kernel7.img': readBlob('usbboot', 'kernel7.img'),
+        'bcm2708-rpi-0-w.dtb': readBlob('usbboot', 'bcm2708-rpi-0-w.dtb'),
+        'bcm2708-rpi-b.dtb': readBlob('usbboot', 'bcm2708-rpi-b.dtb'),
+        'bcm2708-rpi-b-plus.dtb': readBlob('usbboot', 'bcm2708-rpi-b-plus.dtb'),
+        'bcm2708-rpi-cm.dtb': readBlob('usbboot', 'bcm2708-rpi-cm.dtb'),
+        'bcm2709-rpi-2-b.dtb': readBlob('usbboot', 'bcm2709-rpi-2-b.dtb'),
+        'bcm2710-rpi-3-b.dtb': readBlob('usbboot', 'bcm2710-rpi-3-b.dtb'),
+        'bcm2710-rpi-cm3.dtb': readBlob('usbboot', 'bcm2710-rpi-cm3.dtb'),
+        'overlays/dwc2.dtbo': readBlob('usbboot', 'dwc2.dtbo')
```

Comment:
Any reason not to put this `dwc2.dtbo` in `usbboot/overlays/`?
And then instead of always reading in every blob at start-up, you could only read the blobs that usb-boot actually requests on-demand, with only resorting to reading-from-disk for blobs that haven't already been internally cached.

Reply:
Done!